examples/nextjs-next-image-loader/README.md

```diff
@@ -6,54 +6,67 @@ View Demo: <https://cloudinary-nextjs-next-image-loader.netlify.app>
 
 ## 🧰 Using the Cloudinary Loader with the Next.js Image Component
 
-> ⚠️ Note: This is not the  recommended approach. Use the CldImage component with [Next Cloudinary](https://next-cloudinary.spacejelly.dev) for first-class support of Cloudinary with the Next.js Image component.
+> ⚠️ Note: This is not the recommended approach. Use the CldImage component with [Next Cloudinary](https://next.cloudinary.dev) for first-class support of Cloudinary with the Next.js Image component.
 
 To use Cloudinary as a Loader with the Image component, you must specify a `loader` prop for each instance of an Image.
 
-```
-const normalizeSrc = (src) => src[0] === '/' ? src.slice(1) : src
+```js
+function normalizeSrc(src, width) {
+  // Optimization options cannot be taken as arguments in the loader function.
+  // To add optimization options like `q_auto`, `f_auto`, etc., add them to the src URL before the public ID.
+  return `w_${width}/` + src.replace(/^\//, "");
+}
 
-export function cloudinaryLoader({ src, width, quality }) {
-  const params = ['f_auto', 'c_limit', 'w_' + width, 'q_' + (quality || 'auto')];
-  return `https://res.cloudinary.com/${process.env.CLOUDINARY_CLOUD_NAME}/image/upload/${params.join(',')}/${normalizeSrc(src)}`;
+export function cloudinaryLoader({ src, width }) {
```

Comment:
let's use the same example as in the nextjs documentation which is more concise and handles the need, this is what this example was originally based on: 

```
export default function cloudinaryLoader({ src, width, quality }) {
  const params = ['f_auto', 'c_limit', `w_${width}`, `q_${quality || 'auto'}`]
  return `https://example.com/${params.join(',')}${src}`
}
```

https://nextjs.org/docs/app/api-reference/next-config-js/images#cloudinary

the exception is updating the base URL and including the `cloudName` from environment variable, basically the same loader as i had before

the quality is never attempted to be accepted as a string through the loader and is added as a fallback in the parameter construction which i think should remain as the default

```
'q_' + (quality || 'auto')
```

Reply:
ah okay, understood! im guessing previously that was allowed? or given it wasnt TS it just ignored the issue

Reply:
TS would've detected the issue but anyways, I'll replace the string part with a number for the demos it becomes undefined